pages/implementation/start-custom-destination.mdx

```diff
@@ -134,6 +134,45 @@ Call the initAvo function:
 Avo.initAvo({ env: 'dev' }, {}, {}, customDestination);
 ```
 
+#### Javascript to Google Analytics 4
+
+Here is a more specific example of how you can use a custom destination in Javascript on Web to integrate with your Google Analytics 4 SDK to implement custom event tracking with Avo.
+
+```javascript
+import Avo from './Avo';
```

Comment:
Could we include insturctions what this looks like if I'm not using a bundler and just including this on my static page, like we're doing on avo.app?

Reply:
Yes, let's create a separate section on it in our updated docs ✨